### PR TITLE
Contact imports

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by nicpottier** [comment]
Can you explain this a bit more? Is this fixing something? If so, where's the test?

**Comment by norkans7** [comment]
nothing really found yet, just changed this to not have to check the header in many different places, and try to use the same path for all imports

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1042,7 +1045,7 @@ def import_csv(cls, task, log=None):
             try:
                 import_params = json.loads(task.import_params)
             except Exception:
-                pass
+                logger.error("Failed to parse JSON for contact import #d" % task.pk, exc_info=True)
```
Comment (norkans7) [inline temba/contacts/models.py]: Log to sentry we failed to parse JSON

File: temba/contacts/views.py [hunk 2]
```diff
@@ -548,12 +562,8 @@ def post_save(self, task):
...
             org = self.derive_org()
             params = dict(org_id=org.id, timezone=org.timezone, extra_fields=[], original_filename=self.form.cleaned_data['csv_file'].name)
-            task.import_params = json.dumps(params)
-            task.save()
-
-            headers = Contact.get_org_import_file_headers(task.csv_file.file, org)
-            if not headers and not task.done():
-                task.start()
+            params_dump = json.dumps(params)
+            ImportTask.objects.filter(pk=task.pk).update(import_params=params_dump)
```
Comment (norkans7) [inline temba/contacts/views.py]: use queryset update instead of model save method

**Comment by nicpottier** [comment]
That update could be it.. let's try.

**Comment by norkans7** [comment]
sawa

**Pull request merged**